pkgs/tools/networking/rpki-client/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchurl, autoreconfHook, expat, libressl, rsync }:
+
+stdenv.mkDerivation rec {
+  pname = "rpki-client";
+  version = "8.0";
+
+  src = fetchurl {
+    url = "https://cdn.openbsd.org/pub/OpenBSD/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-W3EMzuLn6UlYflTa+COBFnEXSlDGcXRuWidq+qDOVb4=";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
Why are we propagating those? rpki-client looks like a normaly binary so only buildInputs is required and nix does the rest.